Rubberduck.Inspections/Concrete/ProcedureCanBeWrittenAsFunctionInspection.cs

```diff
@@ -46,11 +45,11 @@ public override IEnumerable<IInspectionResult> GetInspectionResults()
                                                   userDeclarations.Where(item => item.IsWithEvents)
                                                                   .All(withEvents => userDeclarations.FindEventProcedures(withEvents) == null) &&
                                                                   !builtinHandlers.Contains(decl))
-                                   .Select(result => new ProcedureCanBeWrittenAsFunctionInspectionResult(
+                                   .Select(result => new InspectionResult(
                                                          this,
-                                                         State,
-                                                         result,
-                                                         new QualifiedContext<VBAParser.SubStmtContext>(result.QualifiedName, (VBAParser.SubStmtContext)result.Context)));                   
+                                                         string.Format(InspectionsUI.ProcedureCanBeWrittenAsFunctionInspectionResultFormat, result.IdentifierName),
+                                                         new QualifiedContext<ParserRuleContext>(result.QualifiedName, (VBAParser.SubStmtContext)result.Context),
+                                                         result));                   
```

Comment:
I almost did that, but then you need to pass `RubberduckParserState` in instead...